examples/data_labeler/data_labeler_predict_train.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "square-burst",
+   "metadata": {},
+   "source": [
+    "# Complex Data Type Detection using Data Labeler component of Data Profiler"
+   ]
+  },
+  {
+   "cell_type": "raw",
```

Comment:
When I pull this up in jupyter, it does not seem to display properly. It looks like all the text is in a codeblock

Reply:
fixed